tests/unittest.sh

```diff
@@ -51,3 +51,7 @@ rm -f unittest_heap_string
 gcc ../ini.c -DINI_CALL_HANDLER_ON_NEW_SECTION=1 unittest.c -o unittest_call_handler_on_new_section
 ./unittest_call_handler_on_new_section > baseline_call_handler_on_new_section.txt
 rm -f unittest_call_handler_on_new_section
+
+gcc ../ini.c -DINI_ALLOW_NO_VALUE=1 unittest.c -o unittest_call_allow_no_value
+./unittest_call_allow_no_value > baseline_call_allow_no_value.txt
+rm -f unittest_call_allow_no_value
```

Comment:
This test and the committed `.txt` file shouldn't have the word "call" in it. Should be called just `unittest_allow_no_value`.